eng/scripts/splittestdependencies/set-artifact-packages.ps1

```diff
@@ -0,0 +1,41 @@
+param (
+    [string] $ProjectNames,
```

Comment:
Why do we need the Project list if we have the PackageInfos?

Reply:
Because the package infos is a static set generated and used by _all_ the test legs. The `Project Names` variable is the set of package names _assigned for this leg_.